volatility3/framework/plugins/windows/thrdscan.py

```diff
@@ -19,7 +21,18 @@ class ThrdScan(interfaces.plugins.PluginInterface, timeliner.TimeLinerInterface)
 
     # version 2.6.0 adds support for scanning for 'Ethread' structures by pool tags
     _required_framework_version = (2, 6, 0)
-    _version = (2, 0, 0)
+    _version = (2, 0, 1)
```

Comment:
I think this is an additive change (ie, people can't rely on the tuple being named, unless it's at least this particular version), so this needs to be a MINOR bump, not just a PATCH bump please.  That also means all the other dependencies (which otherwise wouldn't have been necessary, because internal changes have no impact on dependencies, and the PATCH number isn't even taken account in the verification method).